java/pattern/variable/builder/ValueVariableBuilder.java

```diff
@@ -0,0 +1,20 @@
+package com.vaticle.typeql.lang.pattern.variable.builder;
+
+import com.vaticle.typeql.lang.pattern.constraint.Predicate;
+import com.vaticle.typeql.lang.pattern.constraint.ValueConstraint;
+import com.vaticle.typeql.lang.pattern.variable.ValueVariable;
+
+public interface ValueVariableBuilder extends PredicateBuilder<ValueVariable> {
```

Comment:
if `ThingVariableBuilder` has predicate builder methods on it, why doesn't `ValueVariableBuilder` as well?

Reply:
ThingVariable.Attribute does
I'm not sure the root ThingVariable class has any  methods or how any of that works